<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptTabWidget</class>
 <widget class="QWidget" name="ScriptTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="GUI::L_System::ScriptInfoTabWidget" name="scriptInfoTabs">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="runDepthLabel">
           <property name="text">
            <string>Run to depth:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="runDepthSpinBox"/>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="runScriptButton">
            <property name="text">
             <string>Run Script</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="savedResultsLabel">
             <property name="text">
              <string>Saved Results:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="savedResultsComboBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="saveResultButton">
             <property name="text">
              <string>Save Result</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="discardResultButton">
             <property name="text">
              <string>Discard Result</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="renderSavedResultButton">
             <property name="text">
              <string>Render Saved Result</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QScrollArea" name="savedResultOutput">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="GUI::Output::L_System::ScriptOutputWidget" name="savedResultOutputWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>497</width>
             <height>68</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="GUI::OpenGL::GLWidget" name="openGLWidget"/>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>440</width>
          <height>114</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::OpenGL::GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>GUI/OpenGL/GLWidget.h</header>
  </customwidget>
  <customwidget>
   <class>GUI::L_System::ScriptInfoTabWidget</class>
   <extends>QTabWidget</extends>
   <header>GUI/L_System/ScriptInfoTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GUI::Output::L_System::ScriptOutputWidget</class>
   <extends>QWidget</extends>
   <header>GUI/Output/L_System/ScriptOutputWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
